lib/block-patterns/three-columns-text.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Three columns of text.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Three columns of text', 'Block pattern title', 'default' ),
+	'categories'  => array( 'columns', 'text' ),
+	'content'     => '<!-- wp:columns {"align":"full","style":{"color":{"text":"#000000","background":"#ffffff"}}} -->
+	<div class="wp-block-columns alignfull has-text-color has-background" style="background-color:#ffffff;color:#000000"><!-- wp:column -->
+	<div class="wp-block-column"><!-- wp:heading {"level":3,"style":{"typography":{"fontSize":"24px","lineHeight":"1.3"}}} -->
+	<h3 style="font-size:24px;line-height:1.3"><strong><a href="http://wordpress.org" data-type="URL" data-id="wordpress.org">' . esc_html__( 'Virtual Tour ↗', 'default' ) . '</a></strong></h3>
```

Comment:
what are these data attributes for?

Reply:
I don't know where these came from, removing!